docs/getting-started/local-machine.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started on Local Machine
+
+## Soft Skill Dependencies
+
+Find the person responsible for maintaining the remote machine and ask for the following.
+
+* `REMOTE_MACHINE_IP_OR_HOSTNAME` — remote machine IP or hostname. Examples: `42.42.42.42`, `remote.domain.com`.
+* `REMOTE_MACHINE_PORT` — remote machine port. Example: `42`.
+* `REMOTE_MACHINE_USERNAME` — remote machine user, most likely a personalized, not shared one. Example: `john_doe`.
+
+Think about an alias for the remote machine.
+
+* `REMOTE_MACHINE_ALIAS` — SSH alias for remote machine. Something like `mainframe` or `build-machine`.
+
+## Authentication
+
+1. Generate SSH key and remember the resulting file name as `{SSH_KEY_FILE_NAME}`.
```

Comment:
Mention that it needs to happen on local machine?

Reply:
At this step it is not even possible to do this on a remote machine.